plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties

```diff
@@ -1,2 +1,11 @@
-AppEngineJavaComponentMissing=Cannot create an App Engine Eclipse project because the Cloud SDK App Engine Java component is not installed. Fix by running 'gcloud components install app-engine-java' on the command-line.
-AppEngineLibrariesSelectorGroupLabel=Libraries to add to build path
\ No newline at end of file
+fix.appengine.java.component=Cannot create an App Engine Eclipse project because the\
+ Cloud SDK App Engine Java component is not installed. Fix by running\ 
+ 'gcloud components install app-engine-java' on the command-line.
+fix.cloud.sdk.location=Cannot create an App Engine Eclipse project\
+ because the Cloud SDK App could not be found.\
+ Please install it and set the location in the Google Cloud Platform preferences.
```

Comment:
[We can](http://git.eclipse.org/c/platform/eclipse.platform.swt.git/tree/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet356.java) but I think it becomes a bit complicated.  But since we have a wizard page, we can just put in a standard button.